Tests/LibWeb/AccessibilityBridge/tests/test_attributes.py

```
@@ -0,0 +1,142 @@
+        missing = []
+
+        def visit(obj, _depth):
+            try:
+                role = obj.get_role_name()
+                attrs = get_attributes_dict(obj)
+                if role in ("document web", "filler"):
+                    return
+                if "tag" not in attrs:
+                    missing.append((role, obj.get_name()))
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)